src/function_wrappers.ts

```diff
@@ -202,10 +242,16 @@ const wrapEventFunctionWithCallback = (
  */
 export const wrapUserFunction = <T = unknown>(
   userFunction: HandlerFunction<T>,
-  signatureType: SignatureType
+  signatureType: SignatureType,
+  context?: object
 ): RequestHandler => {
   switch (signatureType) {
     case 'http':
```

Comment:
In the go implementation, we add an `OpenFunction` signature like `function(ctx, data)` https://github.com/OpenFunction/functions-framework-go/blob/main/runtime/runtime.go#L16
And there're totally 3 signature supported:
- HTTP
- CloudEvent
- OpenFunction

The knative sync function support all of the 3 signatures https://github.com/OpenFunction/functions-framework-go/blob/main/runtime/knative/knative.go

While the async function only supports the `OpenFunction` signature https://github.com/OpenFunction/functions-framework-go/blob/main/runtime/async/async.go

Reply:
It's okay to add a new `openfunction` function type.